addons/html_editor/static/tests/chatgpt.test.js

```diff
@@ -0,0 +1,46 @@
+import { expect, test } from "@odoo/hoot";
+import { setupEditor } from "./_helpers/editor";
+import { press, waitFor } from "@odoo/hoot-dom";
+import { animationFrame } from "@odoo/hoot-mock";
+import { contains } from "@web/../tests/web_test_helpers";
+import { insertText } from "./_helpers/user_actions";
+
+import { EXTRA_PLUGINS } from "@html_editor/plugin_sets";
+
+const PROMPT_DIALOG_TITLE = "Generate Text with AI";
+const ALTERNATIVES_DIALOG_TITLE = "AI Copywriter";
+
+test("ChatGPT dialog opens in prompt mode when selection is collapsed (with Powerbox)", async () => {
+    const { editor } = await setupEditor("<p>te[]st</p>", { config: { Plugins: EXTRA_PLUGINS } });
+
+    // Select ChatGPT in the Powerbox.
+    insertText(editor, "/");
+    insertText(editor, "ChatGPT");
+    await animationFrame();
+    press("Enter");
+
+    // Expect the ChatGPT Prompt Dialog to be open.
+    const promptDialogHeaderSelector = `.o_dialog .modal-header:contains("${PROMPT_DIALOG_TITLE}")`
+    await waitFor(promptDialogHeaderSelector);
+    expect(promptDialogHeaderSelector).toHaveCount(1);
+
+    // Expect the ChatGPT Alternatives Dialog not to be open.
+    const alternativesDialogHeaderSelector = `.o_dialog .modal-header:contains("${ALTERNATIVES_DIALOG_TITLE}")`
+    expect(alternativesDialogHeaderSelector).toHaveCount(0);
+});
+
+test("ChatGPT dialog opens in alternatives mode when selection is not collapsed (with toolbar)", async () => {
+    const { editor } = await setupEditor("<p>te[s]t</p>", { config: { Plugins: EXTRA_PLUGINS } });
```

Comment:
Maybe just use MAIN_PLUGIN + chatGPT_Plugin ? You don t need to have all the other extra plugin